test/error_tests.cpp

```diff
@@ -64,7 +64,7 @@ TEST(ErrorTests, BoolFalseTest) {
 TEST(ErrorTests, BoolTrueTest) {
     Error error;
     error.code = ErrorCode::UNSUPPORTED_PROTOCOL;
-    EXPECT_TRUE(error);
```

Comment:
I find the change at this line bothersome, BTW. When I compile _without_ Conan support, the original compiles just fine. _With_ Conan support, the original doesn't. And indeed, I find myself surprised that the original ever compiles at all! Life is full of mysteries.